src/helpers/assets.js

```diff
@@ -230,7 +230,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
     });
   }
 
-  rows = sortBy(rows, ['familyName']);
+  rows = sortBy(rows, row => row.familyName.replace('The ', '').toLowerCase());
```

Comment:
Could use a more general regex to catch all the "the"s: 
```
const theRegex = /^(the|The|THE)\s/g;
rows = sortBy(rows, row => row.familyName.replace(theRegex, '').toLowerCase());
```

Reply:
ty ser 🤝